.github/ISSUE_TEMPLATE.md

```diff
@@ -14,14 +14,23 @@ If it's not a bug, please remove the template and elaborate the issue in your ow
 Steps to reproduce
 ------------------
 
+```powershell
+
+```
 
 Expected behavior
 -----------------
 
+```none
```

Comment:
@SteveL-MSFT: `powershell` is for highlighting PowerShell _source code_, whereas  _output_ will generally _not_ be _source code_ - in which case highlighting as if it _were_ can be distracting.